packages/round-manager/src/features/api/utils.ts

```diff
@@ -176,37 +176,35 @@ export const abbreviateAddress = (address: string) => `${address.slice(0, 8)}...
  * @param metadata - The metadata of a round application
  * @returns The application schema
  */
-export const generateApplicationSchema = (metadata: any) => {
-
-  // declare schema with default fields
-  let schema = [];
-
-  for (const key of Object.keys(metadata)) {
-    if (typeof metadata[key] === "object") {
-
-      for (const subKey of Object.keys(metadata[key])) {
-        schema.push({
-          id: schema.length,
-          question: camelToTitle(subKey),
-          type: "TEXT",
-          required: true,
-          info: metadata[key][subKey],
-          choices: []
-        })
-      }
-      continue
-
-    } else {
-      schema.push({
-        id: schema.length,
-        question: camelToTitle(key),
-        type: "TEXT",
-        required: true,
-        info: metadata[key],
-        choices: []
-      })
-    }
+export const generateApplicationSchema = (metadata: ApplicationMetadata) => {
+  if (!metadata.customQuestions) return []
+
+  const maybe_questions = metadata.customQuestions
```

Comment:
```suggestion
  const customQuestions = metadata.customQuestions
```

let's keep the coding style consistent and keep it camelCase
Additionally let's avoid renaming variables. might cause confusion down the line

Reply:
Ha you are totally right, I have been doing python recently and it slipped in